core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -1288,16 +1288,22 @@ path_get(wasm_exec_env_t exec_env, struct fd_table *curfds,
     size_t expansions = 0;
     char *symlink = NULL;
     size_t symlink_len;
+#ifdef BH_PLATFORM_WINDOWS
+#define PATH_SEPARATORS "/\\"
```

Comment:
Is it a typo, should use `\\`?

Reply:
No, on windows both forward and back slashes are valid path separators e.g.
`dir/nested\\file` should be split into `dir`, `nested` and `file`.